catroid/src/main/java/org/catrobat/catroid/common/Constants.java

```diff
@@ -233,9 +233,9 @@
 	// background sprite is always on index 0
 	public static final int Z_INDEX_BACKGROUND = 0;
 
-	// this offset reflects the offset caused by "virtual" layers (currently only PenActor)
+	// this offset reflects the offset caused by "virtual" layers (currently only PenActor, EmbroideryActor)
 	// which are sneaked in at the Stage creation when starting the scene.
-	public static final int Z_INDEX_NUMBER_VIRTUAL_LAYERS = 1;
+	public static final int Z_INDEX_NUMBER_VIRTUAL_LAYERS = 2;
 
 	// the minimum z index a real sprite layer can have
```

Comment:
also here: the comment does not really explain, why these numbers are added. (I know, it's not your code, but maybe you have a better idea?)

Reply:
removed the unnecessary part but left the first line (it kinda helps a lot to understand for what that constant is for)